hotelReservation/helm-chart/hotelreservation/charts/wrk2/values.yaml

```diff
@@ -0,0 +1,11 @@
+wrk2:
+  replicas: 1
+
+loadgen:
+  numThreads: "2"
+  numConns: "4"
+  duration: "30s"
+  requestsPerSec: 200
+  targetURL: ""
+  scriptPath: "/wrk2/scripts/hotel-reservation/mixed-workload_type_1.lua"
+  replicas: 1  # Number of replicas for the Deployment
```

Comment:
This block was moved here from the overall `hotelreservation/values.yaml`, since it's used only by wrk2.